modules/store/src/selector.ts

```diff
@@ -29,7 +29,7 @@ export interface MemoizedSelector<
 }
 
 /**
- * @deprecated Selectors with props are deprecated, for more info see {@link https://github.com/ngrx/platform/issues/2980 Github Issue}
+ * @deprecated Selectors with props are deprecated, for more info see {@link https://github.com/ngrx/platform/issues/2980 Github Issue} or {@link https://ngrx.io/guide/migration/v12#ngrxstore suggested migration}
```

Comment:
Personally, I don't like having two links here.
I'm fine (and I think it's better) to modify the existing link to the migration page.
What do you think?

```suggestion
 * @deprecated Selectors with props are deprecated, for more info see the {@link https://ngrx.io/guide/migration/v12#ngrxstore migration guide}
```